config/jms/activemq-jms-config.yaml

```diff
@@ -222,4 +222,4 @@ servers:
             name: "[[.KieName]]-amq-jolokia"
 #[[end]]
 ## RANGE ends
-## KIE Servers END
\ No newline at end of file
+## KIE Servers END
```

Comment:
Same here, with newline at end of file, if it's removed

Reply:
@bmozaffa I've tried everything, and no matter what editor I use (vim, gedit, or even online with the github editor), I can't get a newline to persist at the end of this file. Why is it necessary to be there? And if so, what can I do to add it back?

Reply:
Weird, I just submitted a PR against your branch. If you merge it, you can then go back and squash it in and do a push -f?

Without a newline, in some envs you also get a warning when doing a git diff, even when the content is the same near the EOF

Reply:
on IntelliJ I had to put 2 blank lines to count one.